source/components/composing-components.md

```diff
@@ -10,6 +13,8 @@ Just like we compose regular HTML elements, we can do the same with components.
 {{/user-list}}
 ```
 
+First, a review of how to use components.
```

Comment:
I'm not sure the inline vs block distinction is covered anywhere else in the Guides. If that's right, then this isn't a review.